apps/dotcom/client/src/tla/hooks/useTldrFileDrop.ts

```diff
@@ -45,39 +29,11 @@ export function useTldrFileDrop() {
 			if (!tldrawFiles.length) {
 				return
 			}
-
-			const editor = globalEditor.get()
-
-			if (editor) {
-				const snapshots = await getSnapshotsFromDroppedTldrawFiles(editor, tldrawFiles)
-				if (!snapshots.length) return
-
-				const uploadingTitle = intl.formatMessage(messages.uploading, {
-					count: snapshots.length,
-				})
-				const addedTitle = intl.formatMessage(messages.adding, { count: snapshots.length })
-
-				const id = addToast({
-					severity: 'info',
-					title: uploadingTitle,
-				})
-
-				const results = await app.createFilesFromTldrFiles(snapshots, token)
-
-				removeToast(id)
-				addToast({
-					severity: 'success',
-					title: addedTitle,
-					keepOpen: true,
-				})
-				if (results.slugs.length > 0) {
-					navigate(routes.tlaFile(results.slugs[0]), {
-						state: { mode: 'create' } as TlaFileOpenState,
-					})
-				}
-			}
+			app.uploadTldrFiles(tldrawFiles, (file) => {
+				navigate(routes.tlaFile(file.id), { state: { mode: 'create' } satisfies TlaFileOpenState })
```

Comment:
My PR removes the `TlaFileOpenState` completely. Think just removing the state should work here, but let me know.